proto/anpr_camera.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+
+// AnprCameraApi describes the capability to retrieve detection events from ANPR cameras.
+service AnprCameraApi {
+  rpc ListAnprEvents(ListAnprEventsRequest) returns (ListAnprEventsResponse) {}
+  rpc PullAnprEvents(PullAnprEventsRequest) returns (stream PullAnprEventsResponse) {}
+}
+
+// AnprEvent describes a registration plate detection event from an ANPR camera.
+// Includes registration plate information but also vehicle information if the camera supports it.
+message AnprEvent {
+  // The time the detection events occurred.
+  google.protobuf.Timestamp event_time = 1;
+  // The registration plate detected.
+  string registration_plate = 2;
+  // Country of the detected registration plate.
+  string country = 3;
```

Comment:
Could we state that this is e.g. an ISO 3166-1 country code so it's machine-readable?